docs/builtins/Repl/typecheck.md

```diff
@@ -1,52 +1,54 @@
 ## typecheck
 
-Use `(typecheck "my-module")` to run the pact static typechecker on some module `my-module`
+Use `typecheck` to run the Pact static type checker on a specified `module` name.
```

Comment:
This rewrite is a nit. If we want to be technically correct, we're not running the typechecker on a module name, we're running the typechecker on the module named by the first parameter to `typecheck`. That is if your module is named `(module foo ...)`, then if you write `(typecheck "foo")` it will typecheck the `(module foo ..)`, but not just the name `foo`.

I think that the original wording was clearer